src/server/browserType.ts

```diff
@@ -35,20 +35,22 @@ import { CallMetadata, SdkObject } from './instrumentation';
 const ARTIFACTS_FOLDER = path.join(os.tmpdir(), 'playwright-artifacts-');
 
 export abstract class BrowserType extends SdkObject {
-  private _name: registry.BrowserName;
+  private _name: 'chromium'|'firefox'|'webkit';
+  private _binaryName: registry.BrowserName;
   readonly _registry: registry.Registry;
   readonly _playwrightOptions: PlaywrightOptions;
 
-  constructor(browserName: registry.BrowserName, playwrightOptions: PlaywrightOptions) {
+  constructor(name: 'chromium'|'firefox'|'webkit', binaryName: registry.BrowserName, playwrightOptions: PlaywrightOptions) {
     super(playwrightOptions.rootSdkObject, 'browser-type');
     this.attribution.browserType = this;
     this._playwrightOptions = playwrightOptions;
-    this._name = browserName;
+    this._name = name;
+    this._binaryName = binaryName;
```

Comment:
nit: I'm not a huge fan of `_binaryName`. Maybe:
- `_name` -> `_baseBrowserName`
- `_binaryName` -> `_browserName`

Reply:
The word "browser" is very overloaded, so I'd prefer to have distinct names for distinct entities. 

To elaborate: both `//src/utils/registry.ts` and playwright servers `//src/server/browserType.ts` use word "browser" here and there, yet these are two very different things:
- in playwright servers, "browser" means the browser interface we use to communicate. So we have "chromium" for CDP, "webkit" for WebInspector Remote Debugging Protocol, and "firefox" for juggler. These all are "browsers". 
- in registry, "browser" means an entity that we download from our CDN. Originally we were only downloading browsers, but we now have FFMPEG and we are considering moving WinLDD there as well. 

If we were to start over, we'd establish some other terminology for the registry that doesn't include name "browser" - "binary" seems to be fine since it's distinct enough and is not overused around the codebase.